### DISCUSSION TITLE
트랜잭션 전파에 대해서 알아보자~!

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#149] [1. 개발 관련 글 모음]
https://stump-cress-2f5.notion.site/Transactional-852f8174585f4bc0b72e34e87d034c45?pvs=4

오늘도 정말 많이 배웠습니다..!! 감사합니다~

**Comment by shkisme** [comment]
이야... 정리가 👍👍 저 좀 본받아야 할 것 같습니다!
프로젝트 진행하면서 크게 느끼고 있는 점이 있는데, 이렇게 삽질하면서 맞닥뜨린 문제를 한번 해결해보면 머릿속에 엄청엄청 오래 남는다는 거에요....!!! 그냥 책 읽는 것 보다!! 정리까지 남겨놓으면 아주 오래 갈 것 같네요~ 👍👍

**Reply by 02ggang9** [reply]
그러게요!! 트랜잭션 질문 날라오면 바로 대답할 수 있을거 같아요~~
정말 이번 프로젝트에서 배우는 점이 많네요~~ 덕분입니다!! 감사합니다~!!
